content/src/view.js

```diff
@@ -2626,17 +2681,34 @@ function setPaneEditorBlockOptions(pane, pal, modeOptions) {
   }
 }
 
-function setPaneEditorBlockMode(pane, useblocks) {
+function setPaneEditorBlockMode(pane, useblocks, editor) {
+  function setMainEditorBlockMode(editor, useblocks) {
+    if (editor.currentlyUsingBlocks == useblocks) return false;
+    var visibleMimeType = editorMimeType(paneState);
+    if (useblocks && !mimeTypeSupportsBlocks(visibleMimeType)) return false;
+    var togglingSucceeded = editor.toggleBlocks();
+    if (!togglingSucceeded) return false;
+    fireEvent('toggleblocks', [pane, editor.currentlyUsingBlocks]);
+    return true;
+  }
+  function setSubEditorBlockMode(editor, useblocks) {
+    if (!editor) return false;
+    if (editor.currentlyUsingBlocks == useblocks) return false;
+    return editor.toggleBlocks();
+  }
   var paneState = state.pane[pane];
   if (!paneState.dropletEditor) return false;
```

Comment:
It is that way in ~25 places in this file. So, I thought maybe you prefer that for single line conditionals.
Do you want me to all braces for all conditional?